src/routes/users.route.ts

```diff
@@ -0,0 +1,10 @@
+import {createUser, getUsers, getUser, deleteUser} from "../controllers/users.controller";
+import {FastifyInstance, FastifyServerOptions} from "fastify";
+
+export async function routes (fastify: FastifyInstance, options: FastifyServerOptions) {
+    fastify.post('/create', createUser)
+    fastify.get('/', getUsers)
+    fastify.get('/:userId', getUser)

```

Comment:
read about correct positioning of routes with parameters
   
    fastify.get('/:userId', getUser)
    fastify.get('/', getUsers)